### PR TITLE
changed the spelling of introduction

### PR DESCRIPTION
**Pull request by mahakaaal0999**
changed the spelling of introduction

### CODE REVIEW

File: samar.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <div class="main">
             <div class="navbar">
                 <div class="icon">
-                    <h2>Interduction</h2>
+                    <h2>Introduction</h2>
```
Comment (abhideveloper68) [inline samar.html:20]: this is incorrect

**Review by abhideveloper68 — COMMENTED** [review]
Please check my comment

File: samar.html (hunk 1 above)
Comment (mahakaaal0999) [reply samar.html:20]: resolve3d

**Review by abhideveloper68 — APPROVED** [review]
I approved

**Pull request merged**